src/labels/CustomLabels.labels

```diff
@@ -2904,6 +2905,20 @@ To check your GAU Allocation settings, go to Donations | GAU Allocations.</value
         <shortDescription>Message warning user if totals don&apos;t match expected</shortDescription>
         <value>Require Expected Totals Match is selected but the Expected Count of Gifts and Expected Total Batch Amount fields are blank. Edit the batch details and either uncheck Require Expected Totals Match or add values in the related fields.</value>
     </labels>
+    <labels>
+        <fullName>bgeMassDryRun</fullName>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Text for mass dry run in BGE.</shortDescription>
+        <value>Mass Dry Run</value>
+    </labels>
+    <labels>
+        <fullName>bgeMassDryRunProcessing</fullName>
```

Comment:
@bethbrains - Not sure if this the best place for this bit. Feel like it could potentially be used in other places not just the BGE/Mass Dry Run

Reply:
Hi @melissabarber - wanted to check in regarding this new custom label. It's currently used for the toast that pops up when Batch Dry Run is started by the user

Reply:
@psadie Is this with regards to naming it "Mass Dry Run" vs. "Batch Dry Run"?

Reply:
@melissabarber - Not quite, this is regarding a toast that displays once 'Batch Dry Run' is started advising people to not close the page. Currently language is 'Please don't navigate out of this page'.

Reply:
@psadie I'm not quite brave enough to make the edits inline, but I added a couple of comments to the custom labels! Let me know if you have any questions or have further suggestions to help clarify the labels.